src/base/static/components/atoms/buttons.js

```diff
@@ -43,40 +43,21 @@ EditorButton.propTypes = {
   type: PropTypes.string.isRequired,
 };
 
-// TODO: Phase this out as we move to CSS-in-JS:
-const LegacyButton = ({ children, ...props }) => {
+// Influenced by the material-ui api:
+// https://material-ui.com/api/button/
+// NOTE: we may want to consider using material-ui instead
```

Comment:
Does using `material-ui` force us into a Material look and feel, or are we using it just for the UI abstractions?

Reply:
They have a reasonable API, so I decided to base our API off of theirs, for now.

If we started using their components, it would force us into their look and feel. But it would save us the extra work of building out our own atoms here.

Reply:
I think we should discuss the merits of adopting Material's look and feel. Material looks a little corporate to me, and many apps adopt the Material look. I think we have an advantage in our current visual design. But there are tradeoffs, of course, so we should discuss pros and cons.

Reply:
Good point - I'll remove this note, and open an issue about the possibility of adopting a UI framework.